lib/liquid/variable_lookup.rb

```diff
@@ -54,6 +54,8 @@ def evaluate(context)
         elsif @command_flags & (1 << i) != 0 && object.respond_to?(key)
           object = object.send(key).to_liquid
 
+        elsif context.strict_variables
+          raise Liquid::MissingVariable, "missing variable #{key} in #{object}"
```

Comment:
I'm worried about the `in #{object}` part here. That looks like it could potentially leak sensitive information into the rendered output. Can we remove that?